### PR TITLE
Implement winGame and loseGame actions and corrosponding UI

### PR DESCRIPTION
**Pull request by V-rtualized**

### CODE REVIEW

**Comment by V-rtualized** [comment]
The buttons on the win/lose popups are changed to 'Return to Lobby' and 'Leave Lobby', these just call the stop game and leave lobby actions. Maybe we want to do something other than Return to Lobby? Or just have the client return to lobby instead of forcing the whole lobby back? Giving the host the ability to start a rematch would be nice too but I don't think its necessary right now.

**Review by TGMM — APPROVED** [review]

File: Multiplayer/UI/Game_UI.lua [hunk 1]
```diff
@@ -841,6 +1049,355 @@ function Game:start_run(args)
...
+								},
+							},
+						},
+					},
+				},
+				{ n = G.UIT.C, config = { align = "cm", padding = 0.1 }, nodes = { t.nodes[1] } },
+			},
+		}
+
+		--t.nodes[1].config.mid = true
```
Comment (TGMM) [inline Multiplayer/UI/Game_UI.lua:1248]: Maybe you forgot to delete this

File: Multiplayer/UI/Game_UI.lua [hunk 2]
```diff
@@ -841,6 +1049,355 @@ function Game:start_run(args)
...
+								id = "jimbo_spot",
+								object = Moveable(0, 0, G.CARD_W * 1.1, G.CARD_H * 1.1),
+							},
+						},
+					},
+				},
+				{ n = G.UIT.C, config = { align = "cm", padding = 0.1 }, nodes = { t.nodes[1] } },
+			},
+		}
+		--t.nodes[1].config.mid = true
```
Comment (TGMM) [inline Multiplayer/UI/Game_UI.lua:1394]: Same for this

**Review by TGMM — APPROVED** [review]

File: Multiplayer/UI/Game_UI.lua (hunk 1 above)
Comment (V-rtualized) [reply Multiplayer/UI/Game_UI.lua]: This is just a comment that got copy pasted from source code along with the rest of the function

**Pull request merged**